FalmouthFalcons/iteration-3
# Ticket13/jq

Description:
-Fix Given a user has filled out the product form, but not chosen a category
When the user clicks on the Sell button
Then the user should be alerted to select a product category
Images are uploading

Number of Fixes:
- 2

Related Ticket(s):
- Ticket 3, ticket 7

Problem to Solve:
3. User can select product category when selling product #13
7. User can upload a photo of the product #17

Proposed Changes:
-Describe the proposed changes.

Expected Behavior:
- Given a user is authenticated
And performs a gesture on the Sell a product hyperlink
When the product form is rendered
Then there should be a dropdown that displays all product categories

Given a user has filled out the product form, but not chosen a category
When the user clicks on the Sell button
Then the user should be alerted to select a product category

User is able to upload photos.

Steps to Test Solution
1) git fetch
2) git checkout ticket13/jq
3) select blank category and submit, alert should notify you
4) install **brew install imagemagick**
5) Add product and upload photo, photo displays


Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
